_maps/map_files/Mining/Lavaland.dmm

```diff
@@ -3178,7 +3181,7 @@
 /obj/structure/barricade/wooden/crude,
 /obj/machinery/door/airlock/research{
 	name = "Robotics Lab";
-	req_access_txt = "29"
+	req_access_txt = "47,54,29"
```

Comment:
Are these changes intentional? (Also should be ; instead of , iirc)

Reply:
i looked at other doors and i think this is how they work?